src/model_scoring_functions.R

```diff
@@ -121,7 +98,7 @@ process_target_data <- function(hub_path = here::here(),
 #' @return Returns a data frame containing energy scores by location and date
 calc_energy_scores <- function(targets, df_model_output){
   # Energy Scores
-  columns <- c("energy", "brier", "location", "target_date")
+  columns <- c("energy", "brier_point", "brier_dist", "location", "target_date", "scored")
   df_scores <- data.frame(matrix(nrow = 0, ncol = length(columns)))
```

Comment:
two options:
1. pre-define the entire data.frame df_scores with number of rows expected, then keep track of index and fill in one row at a time.
2. use purrr::map_dfr() which would basically achieve the same thing...

Reply:
Noting similar problem with object `samp_multinomial_counts` below (line 166). Might not be as big of an issue with a matrix though.